node/nodejs_intro/README.md

```diff
@@ -140,23 +196,29 @@ What is expected output?
     Yoda
     I am
 </details>
```

Comment:
I really like that you've tucked the results in a summary/details tag. I think it's great to give learners a moment to think/hypothesize before seeing the results